packages/wallet-sdk/src/models/profile.ts

```diff
@@ -88,7 +88,7 @@ export const fetchAccountProfileUrl = async ({
 };
 
 export function signProfileForUpload({ profile, account }: { profile: Profile; account: Account }) {
-  const privateKey = account.dataPrivateKey;
+  const privateKey = account.stxPrivateKey.slice(0, 64);
```

Comment:
What is the relevance of 64 here?
```suggestion
  const descriptiveNameOfWhatNumberIs = 64
  const privateKey = account.stxPrivateKey.slice(0, descriptiveNameOfWhatNumberIs);
```

Reply:
@kyranjamie Is the part required for compressed public key. 
There are 8 places in the repo that uses this expression `slice(0, 64)`. Should I replace all of them? Might be better as a separate PR.